doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -76,453 +76,453 @@ Each of the three main sections own sets of properties:
-|                                   | and the value entered is 1001,    |
-|                                   | the field is displayed as         |
-|                                   | `1001 <http://emsearch.rutgers.ed |
-|                                   | u/atlas/1001_summary.html>`__     |
-|                                   | (hyperlinked to                   |
-|                                   | \http://emsearch.rutgers.edu/atlas|
-|                                   | /1001_summary.html)).             |
-+-----------------------------------+-----------------------------------+
-| <img src="#VALUE" alt="#NAME"     | For displaying the image of an    |
-| class="metadata-logo"/><br/>      | entered image URL (used to        |
-|                                   | display images in the producer    |
-|                                   | and distributor logos metadata    |
-|                                   | fields).                          |
-+-----------------------------------+-----------------------------------+
-| #VALUE:                           | Appends and/or prepends           |
-|                                   | characters to the value of the    |
-| \- #VALUE:                        | field. e.g. if the displayFormat  |
-|                                   | for the distributorAffiliation is |
-| (#VALUE)                          | (#VALUE) (wrapped with parens)    |
-|                                   | and the value entered             |
-|                                   | is University of North            |
-|                                   | Carolina, the field is displayed  |
-|                                   | in the UI as (University of       |
-|                                   | North Carolina).                  |
-+-----------------------------------+-----------------------------------+
-|    ;                              | Displays the character (e.g.      |
-|                                   | semicolon, comma) between the     |
-|    :                              | values of fields within           |
-|                                   | compound fields. For example,     |
-|    ,                              | if the displayFormat for the      |
-|                                   | compound field “series” is a      |
-|                                   | colon, and if the value           |
-|                                   | entered for seriesName is         |
-|                                   | IMPs and for                      |
-|                                   | seriesInformation is A            |
-|                                   | collection of NMR data, the       |
-|                                   | compound field is displayed in    |
-|                                   | the UI as IMPs: A                 |
-|                                   | collection of NMR data.           |
-+-----------------------------------+-----------------------------------+
```

Comment:
Are the changes to the tables above meaningful? Is the content or look changing? Or can these changes simply be ignored?

Reply:
No changes intended. It looks like Eclipse tried to do some cleanup. That might be good but I see in one case (the fieldType param for fields) it added a new column to the table, so I'll just revert all of it since the existing stuff works.